src/main/java/net/william278/velocitab/commands/VelocitabCommand.java

```diff
@@ -0,0 +1,67 @@
+package net.william278.velocitab.commands;
+
+import com.velocitypowered.api.command.CommandSource;
+import com.velocitypowered.api.command.SimpleCommand;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.TextColor;
+import net.william278.desertwell.AboutMenu;
+import net.william278.desertwell.Version;
+import net.william278.velocitab.Velocitab;
+
+import java.util.List;
+
+public class VelocitabCommand implements SimpleCommand {
+    private final AboutMenu aboutMenu;
+    private final Velocitab plugin;
+
+    public VelocitabCommand(Velocitab plugin) {
+        this.plugin = plugin;
+        aboutMenu = AboutMenu.create("Velocitab")
```

Comment:
```suggestion
        this.aboutMenu = AboutMenu.create("Velocitab")
```
super minor nit, but I prefer declaring `this` when settings member variables in a constructor